hasher-matcher-actioner/webapp/src/Settings.jsx

```diff
@@ -5,13 +5,28 @@
 import React from 'react';
 import Tab from 'react-bootstrap/Tab';
 import Tabs from 'react-bootstrap/Tabs';
+import {useHistory, useParams} from 'react-router-dom';
 import ActionSettingsTab from './settings/ActionSettingsTab';
 import SignalSettingsTab from './settings/SignalSettingsTab';
 
 export default function Settings() {
+  const {tab} = useParams();
+  const history = useHistory();
+  if (
+    tab === undefined ||
+    !tab ||
+    (tab !== 'signals' && tab !== 'pipeline' && tab !== 'actions')
+  ) {
+    window.location = '/settings/signals';
+  }
```

Comment:
Wouldn't [`defaultActiveKey`](https://react-bootstrap.github.io/components/tabs/#tabs-api) in <Tabs> solve this issue?

Reply:
as discussed this was part of the implementation to make tabs have routes.